common/util.go

```diff
@@ -382,3 +382,15 @@ func DoWithOverrideReadOnlyOnAzureFiles(ctx context.Context, action func() (inte
 	_, err = action()
 	return err
 }
+
+// @brief Thia API check if the container name provided is a system container or not
+func IsSystemContainer(containerName string) bool {
+	// define the system variables for the system containers
+	systemContainers := []string{"$blobchangefeed", "$logs"}
```

Comment:
These containers as destination of copy shall be errored out but as source it shall still work. Add test cases around such combinations in E2E